llvm/lib/Target/Xtensa/XtensaSubtarget.h

```diff
@@ -39,6 +39,9 @@ class XtensaSubtarget : public XtensaGenSubtargetInfo {
   // Enabled Xtensa Density extension
   bool HasDensity;
 
+  // Enabled Xtensa Windowed Register Option
```

Comment:
s/Option/Feature/?

Reply:
@arsenm , may I ask you for advice, what naming scheme should be used for Xtensa extensions( or options)?
Like @bugadani noted ,the Xtensa architecture have some ISA Options (in other words standard extensions) in documentation. It is differs a bit from for example RISCV terminology and other architectures, the documentation for  RISCV architecture usually use "ISA extension" phrase is such cases. The Xtensa architecture allows to define user specific ISA and such case it is named TIE extensions (TIE language).